client/src/components/PinkHeader/PinkHeader.css

```diff
@@ -0,0 +1,63 @@
+*{
+    font-family: var(--default-font)
+}
+
+.PinkHeader {
```

Comment:
Can you come up with a more abstract name? Not all the headers are pink

Reply:
Yeup